x-pack/plugins/ml/public/application/jobs/new_job/pages/components/pick_fields_step/components/categorization_view/category_stopped_partitions.tsx

```diff
@@ -0,0 +1,118 @@
+import { CategorizationJobCreator } from '../../../../../common/job_creator';
+import { Results } from '../../../../../common/results_loader';
+import { ml } from '../../../../../../../services/ml_api_service';
+
+const NUMBER_OF_PREVIEW = 5;
+export const CategoryStoppedPartitions: FC = () => {
+  const { jobCreator: jc, resultsLoader } = useContext(JobCreatorContext);
+  const jobCreator = jc as CategorizationJobCreator;
+  const [tableRow, setTableRow] = useState<Array<{ count?: number; example: string }>>([]);
+  const [hasStoppedPartitions, setHasStoppedPartitions] = useState(false);
+
+  let _resultsSubscription;
+
+  const columns = useMemo(
+    () => [
+      {
+        field: 'partitionName',
+        name: i18n.translate(
+          'xpack.ml.newJob.wizard.pickFieldsStep.stoppedPartitionsPreviewColumnName',
+          {
+            defaultMessage: 'Stopped partition name',
+          }
+        ),
+        render: (partition: any) => (
+          <EuiText size="s">
+            <code>{partition}</code>
+          </EuiText>
+        ),
+      },
+    ],
+    []
+  );
+  function setResultsWrapper(results: Results) {
+    loadCategoryStoppedPartitions();
+  }
+
+  async function loadCategoryStoppedPartitions() {
+    const results = await ml.results.getCategoryStoppedPartitions([jobCreator.jobId]);
+
+    if (
+      results?.jobs !== undefined &&
+      Array.isArray(results?.jobs[jobCreator.jobId]) &&
+      results.jobs[jobCreator.jobId].length > 0
+    ) {
+      const stoppedPartitionsPreview = results.jobs[jobCreator.jobId];
+      // once we have reached number of stopped partitions we wanted to show as preview
+      // no need to keep fetching anymore
+      if (
+        stoppedPartitionsPreview.length >= NUMBER_OF_PREVIEW &&
+        _resultsSubscription !== undefined
+      ) {
+        _resultsSubscription.unsubscribe();
+        _resultsSubscription = undefined;
+      }
+      setHasStoppedPartitions(true);
+      setTableRow(
+        stoppedPartitionsPreview.slice(0, NUMBER_OF_PREVIEW).map((partitionName) => ({
+          partitionName,
+        }))
+      );
+    }
+  }
+
+  useEffect(() => {
+    // only need to run this check if jobCreator.perPartitionStopOnWarn is turned on
+    if (jobCreator.perPartitionCategorization && jobCreator.perPartitionStopOnWarn) {
+      // subscribe to result updates
+      _resultsSubscription = resultsLoader.subscribeToResults(setResultsWrapper);
+      return () => {
+        if (_resultsSubscription) {
+          _resultsSubscription.unsubscribe();
+        }
+      };
+    }
+  }, []);
+
+  return (
+    <>
+      {hasStoppedPartitions && (
+        <>
+          <EuiSpacer />
+          <div>
+            <FormattedMessage
+              id="xpack.ml.newJob.wizard.pickFieldsStep.categorizationStoppedPartitionsTitle"
+              defaultMessage="Stopped partitions"
+            />
+          </div>
+
+          <EuiCallOut
```

Comment:
How does this look with a small `EuiSpacer` between the 'Stopped partitions' heading and the callout?

Reply:
Looks good! Added the small spacer here: https://github.com/elastic/kibana/pull/75709/commits/618fa8fab2d16c4406f007e3d8c367b7c1ad5795
<img width="1077" alt="Screen Shot 2020-08-24 at 4 16 15 PM" src="https://user-images.githubusercontent.com/43350163/91097352-278dbc00-e625-11ea-8188-2fd5158246cd.png">